README.md

```diff
@@ -517,26 +517,28 @@ Please let us know on [the issue tracker](https://github.com/google/jax/issues)
 if you run into any errors or problems with the prebuilt wheels.
 
 ### pip installation: Google Cloud TPU
-JAX also provides pre-built wheels for
+
+JAX provides pre-built wheels for
 [Google Cloud TPU](https://cloud.google.com/tpu/docs/users-guide-tpu-vm).
 To install JAX along with appropriate versions of `jaxlib` and `libtpu`, you can run
 the following in your cloud TPU VM:
 ```bash
 pip install jax[tpu] -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
 ```
 
-### pip installation: Colab TPU
```

Comment:
One benefit of this section is that without it many users may not realize that "Google Cloud TPU" and "Colab TPU" are different things that require different installation processes. Until Colab drops its deprecated TPU support entirely, I'd be in favor of keeping this section here.

Reply:
Reworded per discussion.